src/tagmanager/tm_parser.h

```diff
@@ -78,7 +78,6 @@ enum
 	TM_PARSER_SH,
 	TM_PARSER_D,
 	TM_PARSER_FORTRAN,
-	TM_PARSER_FERITE,
```

Comment:
I just noticed it's not probably OK to do this - it would shift parser identifiers for plugins. We could either keep the dummy `TM_PARSER_FERITE` ID here and instead of `FeriteParser` use whatever other parser in `tm_parsers.h` or replace it with some newly introduced parser (e.g. the GDScript parser or Perl6 parser which I think should be a filetype Geany should add support for).

Reply:
Good catch, `TM_UNUSED1` perhaps?